src/App.js

```diff
@@ -1,10 +1,27 @@
-import logo from './logo.svg';
+import { Routes, Route } from "react-router-dom";
 import './App.css';
 
+import {Home} from "./pages/home/home.jsx";
+import {Cart} from "./pages/cart/cart.jsx";
+import {WishList} from "./pages/wish-list/wish-list.jsx";
+import {ProductListing} from "./pages/product-listing/product-listing.jsx";
+import {Nav} from "./components/nav.jsx";
```

Comment:
you can create an index file and export it there. here you can import in a single line.